titus-server-runtime/src/main/java/com/netflix/titus/runtime/endpoint/admission/JobImageSanitizer.java

```diff
@@ -49,62 +52,49 @@ public JobImageSanitizer(JobImageValidatorConfiguration configuration, RegistryC
         this.validatorMetrics = new ValidatorMetrics(this.getClass().getSimpleName(), spectatorRegistry);
     }
 
+    /**
+     * @return sanitized Image or {@link Optional#empty()} when sanitization was skipped
```

Comment:
The above IAM `sanitize()` comment is specifying that a function is being returned. Should this say the same?

Reply:
yes, good catch. I will fix.